services/audio/src/main/resources/application.yml

```diff
@@ -7,9 +7,6 @@ spring:
   main:
     banner-mode: "off"
 
-  profiles:
```

Comment:
Put this back in please.  If you've rebased it'll work now.

Reply:
Will do.